src/pybind/mgr/cephadm/module.py

```diff
@@ -2650,6 +2664,8 @@ def get_daemon_names(daemons: List[str]) -> List[str]:
                 deps = [self.iscsi_service.get_trusted_ips(iscsi_spec)]
             else:
                 deps = [self.get_mgr_ip()]
+        elif daemon_type == 'nvmeof':
+            deps = []  # TODO(redo)
```

Comment:
if we don't think we have any deps I think we can just leave this case out. There's already a bunch of other types e.g. `mgr`, `mon`, `loki`, `rgw` that don't have cases here because they have no deps

Reply:
Agree, I removed the if branch as nothing is being done